### PR TITLE
gh-106217: Truncate the issue body size of `new-bugs-announce-notifier`

### PR DESCRIPTION
**Pull request by sobolevn**
* Issue: gh-106217

### CODE REVIEW

File: .github/workflows/new-bugs-announce-notifier.yml [hunk 1]
```diff
@@ -40,7 +40,10 @@ jobs:
                 url    : issue.data.html_url,
                 labels : issue.data.labels.map(label => { return label.name }).join(", "),
                 assignee : issue.data.assignees.map(assignee => { return assignee.login }),
-                body   : issue.data.body
+                // We need to trunkate the body size, because the max size for
```
Comment (hugovk) [inline .github/workflows/new-bugs-announce-notifier.yml]:
```suggestion
                // We need to truncate the body size, because the max size for
```

**Review by hugovk — APPROVED** [review]
See also https://github.com/python/core-workflow/issues/456.

**Comment by jkloth** [comment]
It seems that now there is no body for any message sent via the notifier after this change.  See the [new-bugs-announce](https://mail.python.org/pipermail/new-bugs-announce/2023-July/date.html) archives for issue 106333 (has body) vs 106334 (no body)

**Comment by hugovk** [comment]
Aha, looking at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring, the args are:

```js
substring(indexStart)
substring(indexStart, indexEnd)
```

> `indexStart`
> The index of the first character to include in the returned substring.
> 
> `indexEnd` Optional
> The index of the first character to exclude from the returned substring.


So:
```js
issue.data.body.substring(8000)
```
is attempting to get the substring from char number 8000 up to the end, meaning we lose the first 8000 chars, and get an empty string if there's nothing beyond that. For example:


```js
const str = 'Mozilla';
console.log(str.substring(4));
// prints "lla"
console.log(str.substring(8000));
// prints ""
```

Instead we want something like `issue.data.body.substring(0, 8000)` for the substring from 0 up to 8000.

```js
console.log(str.substring(0, 4));
// "Mozi"
console.log(str.substring(0, 8000));
// "Mozilla"
```

Does that look right? Would you like to make a new PR?

**Comment by sobolevn** [comment]
Oh my, js strikes again 🤦

**Pull request merged**